manifest-production.yml

```diff
@@ -4,7 +4,7 @@ inherit: manifest-base.yml
 
 routes:
   - route: managing-registers-production.cloudapps.digital
-  - route: managing-registers.cloudapps.digital
+  - route: manage.registers.service.gov.uk
```

Comment:
could we just delete `managing-registers-production.cloudapps.digital`? not sure this is serving any purpose?

Reply:
I agree I don't think we need it. Shall I just delete or wait for @danmitchell- to confirm?